openfl/federated/task/runner_keras.py

```diff
@@ -169,6 +169,15 @@ def train_task(
 
         return global_tensor_dict, local_tensor_dict
 
+    def _initialize_metrics_result(self,kwargs):
+        if "batch_size" in kwargs:
+            batch_size = kwargs["batch_size"]
+        else:
+            batch_size = 1
```

Comment:
Simplification: `batch_size = kwargs.get("batch_size", 1)`

Reply:
Thanks for the suggestion - fixed